lib/fluent/plugin/in_tail.rb

```diff
@@ -258,10 +258,15 @@ def receive_lines(lines, tail_watcher)
               end
         begin
           router.emit_stream(tag, es)
+        rescue BufferQueueLimitError
+          return false
         rescue
-          # ignore errors. Engine shows logs and backtraces.
+          # ignore non BufferQueueLimitError errors because in_tail can't recover. Engine shows logs and backtraces.
+          return true
         end
       end
+
+      return true
```

Comment:
true/false is ambiguous what it means. please write a yardoc comment for the return value.